tests/debugger/bug01753-SkeletonPage.inc

```diff
@@ -9,4 +9,4 @@ class SkeletonPage
 	protected function func()
 	{
 	}
-}
\ No newline at end of file
+}
```

Comment:
This seems an unrelated change, and hence don't add it to the commits in this PR.

Reply:
I opened https://github.com/xdebug/xdebug/pull/605 for it, I will then rebase this PR